app/src/main/java/com/worldsnas/starplayer/view/musics_list/MusicsListFragment.kt

```diff
@@ -67,12 +72,20 @@ class MusicsListFragment : Fragment() {
         val musicObserver = Observer<List<MusicRepoModel>> {
 
             val musics = it
+            prepareMusicListService(musics)
```

Comment:
kolle list ro pass midim be service?
ke chi beshe? age gharare kolesh pakhsh beshe khob khode service mitone az Repo begire dige nemitone?

Reply:
zamani bayad list ro pass bedim be  service ke user ru ye music click kone

Reply:
soal e man chize dige i bod.
man naporsidam key pas midim goftam chera pass midim

Reply:
khob age bekhaim kole ahanghaye list pakhsh bayad kolesh ro pass bedim be service.
age rahe dige hast pishnahad bedin. merci